test/remote/gateways/remote_reach_test.rb

```diff
@@ -202,6 +215,17 @@ def test_successful_refund
     assert response.params['response']['RefundId'].present?
   end
 
+  def test_successful_refund
+    response = @gateway.refund(
+      @amount,
+      '5cd04b6a-7189-4a71-a335-faea4de9e11d',
+      { order_id: 'REFUND_TAG' }
+    )
+
+    assert_success response
+    assert response.params['response']['RefundId'].present?
+  end
```

Comment:
Both test **test_successful_refund_with_reference_id** and **test_successful_refund** are the same, maybe you can remove the second one and preserve the most complete `def` description

Reply:
oh now I see the difference, it is about the `order_id` and `reference_id`, then NVM